### ISSUE TITLE
Need a documentation or guidance

### ISSUE THREAD

**Issue by frankiedrake** [vllm-project/vllm#1470]
I want to process a pool of data via the model engine on a machine with a multiple GPUs. Say a 100'000 items. 
I start a model engine with a `llm = LLM(model, tensor_parallel_size=gpus,)`. 
When I pass my data to a `llm.generate(messages, params)` it starts processing my prompts and printing the progress.

So the questions are: 
- sometimes processing hanges with a mistery error `Double free...smthng` (when I encounter it again, I can add full stacktrace), but if I call generate(prompt) again, I can see that this new request is added to a queue, so my previous requests are still processing (? waiting?), how can I trigger the processing again?
- how can I get already processed requests output? I found a method `llm.llm_engine.step()` - which seems to return already processed calls result, but what if processing continues after it?

To summarize. It would be great to have a mechanism or the documentation on how to: 
- trigger engine to start processing awaiting requests
- get already processed prompts with its output
- get waiting requests
- abort all requests

Sorry If I missed this information somewhere
